railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -231,6 +231,24 @@ def initialize(*args)
           raise Error, "Invalid value for --database option. Supported for preconfiguration are: #{DATABASES.join(", ")}."
         end
 
+        # Fail hard if node or yarn isn't detected on the system
+        if options[:webpack]
+          check_node
+          check_yarn
+        end
+
+        # --webpack uses webpacker, which uses action view helpers
+        # --api skips action view
+        if options[:webpack] && options[:api]
```

Comment:
It is possible to use, but there is no out of the box support like helpers to link those webpack bundles. If someone wants to add this they have to install HTML plugin and add some changes to make this work